omero/sysadmins/unix/server-install-homebrew.rst

```diff
@@ -41,175 +51,280 @@ running::
 
+5. Once the build completes, the OMERO server build output will be located in :file:`~/Projects/Omero/code/openmicroscopy/dist`.
+   If it is not already open, open your :file:`.bash_profie`::
+
+    $ open -a TextEdit.app ~/.bash_profile
+
+   Prepend the :file:`bin` directory to your :envvar:`PATH`::
+
+    export PATH=~/Projects/Omero/code/openmicroscopy/dist/bin:$PATH
+
+   Using the command-line terminal, reload your :file:`.bash_profile` using::
+
+    $ source ~/.bash_profile
+
+   To ensure OMERO is correctly linked into your OS X :envvar:`PATH`, type the following in terminal and ensure
+   you get a similar output::
+
+    $ which omero
+    /Projects/omero/code/openmicroscopy/dist/bin/omero
+
+6. Activate the Virtualenv enviornment that we created earlier in the "Requirements"
+   section::
+
+    $ source ~/Virtual/Omero/bin/activate
+
+7. Install Python dependencies using pip::
+
+    $ pip install -r ~/Projects/omero/code/openmicroscopy/dist/share/web/requirements-py27-all.txt
+
+
+OMERO configuration
+-------------------
+
+1. From a fresh command-line terminal, start the database server::
+
+    $ pg_ctl -D /usr/local/var/postgres -l /usr/local/var/postgres/server.log -w start
+
+  .. note::
+   **(Optional)** To make life easier, you can add an ```alias``` to your :file:`.bash_profile`::
+    
+    # Start Virtualenv for OMERO
+    alias startVmOmero="source ~/Virtual/Omero/bin/activate"
+
+   You can also add an `alias` to start and stop the Postgres service::
 
+    alias startPg='pg_ctl -D /usr/local/var/postgres -l /usr/local/var/postgres/server.log -w start'
+    alias stopPg='pg_ctl -D /usr/local/var/postgres -l /usr/local/var/postgres/server.log -w stop'
 
-Then prepend the development :file:`bin` directory to your :envvar:`PATH` to
-pick the right executbale::
+   Reload :file:`.bash_profile` in OS X::
 
-    $ export PATH=~/code/projects/openmicroscopy/dist/bin:$PATH
+    $ source ~/.bash_profile
 
-and follow the steps for setting up the database and OMERO data directory as mentioned in the previous section.
+2. To use Omero, we need to first setup Postgres. Open a command-line terminal and run the 
+   following commands to create a user called *db_user* and database called *omero_database*::
+
+    $ createuser -w -D -R -S db_user
+    $ createdb -E UTF8 -O db_user omero_database
+
+3. Create directory for OMERO to store its data::
+
+    $ mkdir -p ~/Projects/Omero/data
+
+4. Start your Virtualenv enviornment we created earlier::
+
+    $ source ~/Virtual/omero/bin/activate
+
+5. Now set the OMERO configuration::
+
+    $ omero config set omero.data.dir ~/Projects/Omero/data
+    $ omero config set omero.db.name omero_database
+    $ omero config set omero.db.user db_user
+    $ omero config set omero.db.pass db_password
+
+6. Create and run script to initialize the OMERO database::
+
+    $ omero db script --password omero -f - | psql -h localhost -U db_user omero_database
```

Comment:
@rgozim : You wanted to remove the OMERO.web section below, as it is not the development server default and confuses matters.

Reply:
But for that, we need to add to the config above the lines dealing with web server being develop, see